ui_sim.py

```diff
@@ -12,6 +12,8 @@
 
```

Comment:
It appears that the game doesn't load after hitting start game:
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/123680493/46055414-3a83-4301-a74d-261cc5393cec)

Reply:
Here are the errors I get:
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/123680493/41318c4f-4ec7-4845-bc15-6fc96ae5cda4)

Reply:
Sophie confirmed it is in fact a branch issue.

Reply:
Fixed.